modules/knative-eventing-CR-system-deployments.adoc

```diff
@@ -31,6 +32,10 @@ metadata:
   namespace: knative-eventing
 spec:
   deployments:
+  - name: eventing-controller
```

Comment:
There's already `- name: eventing-controller` a few lines below so I think we should just add the `readinessProbes` configuration there. The rest of this PR looks ok.